src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -95,6 +96,19 @@ public static function getExtras()
         return $extras;
     }
 
+    public static function getActiveCapabilities(): array
+    {
+        $activeCapabilities = [];
+        if (
+            !EventsHandler::isEventsDisabled('payments') &&
+            'yes' === StoreKeeperOptions::get(StoreKeeperOptions::PAYMENT_GATEWAY_ACTIVATED, 'yes')
```

Comment:
```suggestion
           StoreKeeperOptions::isPaymentsEnabled()
```

Reply:
Changed into positive checking and moved functions to `StorekeeperOptions`.